src/main/java/seedu/address/ui/ButtonsPanel.java

```diff
@@ -0,0 +1,56 @@
+package seedu.address.ui;
```

Comment:
these UI changes can't be used using a CLI. So, they don't earn credit. However, I will still consider this as a contribution for v1.1 milestone